notifications_utils/polygons.py

```diff
@@ -108,32 +138,32 @@ def buffer_outward_in_degrees(self):
             # broadcast then this joins them together. The aim is to
             # reduce the total number of polygons in areas with many
             # small shapes like Orkney or the Isles of Scilly.
-            self.approx_bleed_in_degrees / 3
+            self.approx_bleed_in_m / 3
         ) + (
             self.perimeter_length / self.perimeter_to_buffer_ratio
         )
 
     @cached_property
-    def buffer_inward_in_degrees(self):
+    def buffer_inward_in_m(self):
```

Comment:
same here about docstring

Reply:
Fixed and squashed into 2da2fa78cd570fd86a46f021853021abcbe5788b